src/main/resources/api.yaml

```diff
@@ -585,22 +585,28 @@ definitions:
   SprintSummaryDto:
     type: object
     properties:
+      sprint:
+        $ref: '#/definitions/SprintDto'
       membersAvailability:
         type: array
         items:
           $ref: '#/definitions/UserAvailabilityDto'
         description: List of members' availability for given sprint
-      totalAvailableTime:
-        type: integer
-        format: int64
-        description: In minutes
-      sprintCoefficient:
+      prevHistoricalSprintCoefficient:
+        type: number
+        format: double
+      thisSprintCoefficient:
```

Comment:
Jak dla mnie `this` -> `current`

Reply:
to nie musi byc current

Reply:
Hmm, bardziej miałam na myśli "current" w sensie "aktualnie przeglądany". 

A w sumie tak naprawdę nazwa `sprintCoefficient` powinna być wystarczająca, szczególnie jak pozostałe pola mają w nazwie "average"

Reply:
Bo potem skończymy z `this.thisSprintCoefficient` w konstruktorze :stuck_out_tongue: